src/09-word-problems.js

```diff
@@ -127,7 +162,33 @@ function compareLocations(address1, address2) {}
     //>   },
     //> ];
  */
-function gradeAssignments(assignments) {}
+function gradeAssignments(assignments) {
```

Comment:
Nice work on this function!

It's got some complicated nested logic, so I think you could benefit by splitting up your work into smaller sub problems using helper functions. Could you try refactoring?